strawberryfields/gbs/sample.py

```diff
@@ -371,3 +170,16 @@ def modes_from_counts(s: list) -> list:
     for i, c in enumerate(s):
         modes += [i] * c
     return sorted(modes)
+
+
+def postselect(samples: list, min_count: int) -> list:
```

Comment:
This should have a `min_count` and also a `max_count` to be as general as possible. For example, this would allow postselecting on a specific photon number

Reply:
Nice! Have added these in!